src/components/pages/board/BoardHeader.tsx

```diff
@@ -1,25 +1,37 @@
-import React from 'react'
-import { Link } from 'react-router-dom'
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
 import { ChevronDown } from 'lucide-react';
+import { fetchBoardTitle } from '../../../utils/boardUtils';
 
-interface BoardHeadderProps {
-    title: string;
+interface BoardHeaderProps {
+  boardId: string | undefined;
```

Comment:
동일 코드인데 같은 의미라 짧게 하는거 추천드려용!

```suggestion
  boardId?: string;
```